packages/devtools-network-console/src/reducers/websocket/index.ts

```diff
@@ -96,6 +84,19 @@ export default function reduceWebsocket(collection: WS_State = DEFAULT_WS_STATE,
         };
         return collection.set(reqId, state);
     }
+    if (action.type === 'REQUEST_WEBSOCKET_CLEAR_MESSAGES') {
```

Comment:
You previously had left a TODO here to convert this to a `switch`. I think the time has come :)

Reply:
for sure. Done with the follow up PR and merged into this one